cocos/editor-support/spine/SkeletonAnimation.cpp

```diff
@@ -40,6 +40,9 @@ using std::vector;
 
 namespace spine {
 
+static std::unordered_map<std::string, spSkeletonData*> skeletonDataMap;
+static std::vector<spAtlas*> atlases;
```

Comment:
I think better to create a skeleton cache file structure to hold both spSkeletonData* and spAtlas*.
In this case only one map collection will be.
Pros:
- easier deleting (in only one for)
- easier to delete only one element if in future it will be needed